packages/cli/lib/lib/babel-config.js

```diff
@@ -27,11 +27,11 @@ module.exports = function (env, options={}) {
 			require.resolve('babel-plugin-transform-react-constant-elements'),
 			isProd && require.resolve('babel-plugin-transform-react-remove-prop-types'),
 			[require.resolve('babel-plugin-transform-react-jsx'), { pragma: 'h' }],
-			[require.resolve('babel-plugin-jsx-pragmatic'), {
-				module: 'preact',
-				export: 'h',
-				import: 'h'
-			}]
+			// [require.resolve('babel-plugin-jsx-pragmatic'), {
```

Comment:
Is this commented out by mistake?

Reply:
no, intentional, webpack was throwing errors about `h` already being defined, so I used the webpack provide plugin instead